### ISSUE TITLE
Remove horizontal scrolling from content listing

### ISSUE THREAD

**Issue by irfansharif** [cockroachdb/docs#6761]
Most pages on our docs have horizontal scrolling, which we should do without. We already re-size/fold content based on window width, horizontal scrolling on top of that adds the following clipping effect.

![image](https://user-images.githubusercontent.com/10536690/75900692-4dbb6780-5e0b-11ea-88a6-6fc2bcc9f9ff.png)

**Comment by jseldess** [comment]
@irfansharif, I just remembered that some pages required horizontal scrolling, for example, pages with broader SQL diagrams: https://www.cockroachlabs.com/docs/stable/configure-zone.html#synopsis.

I think the real fix is removing a horizontal scrollbar when no content exceeds the main body frame of the page. Does that sound reasonable to you?

**Comment by irfansharif** [comment]
>  SQL diagrams: https://www.cockroachlabs.com/docs/stable/configure-zone.html#synopsis.

IMO just the components that are that broad should have horizontal scroll, not the entire page.

**Comment by jseldess** [comment]
Good point. That should be possible. @anagio, do you think you could squeeze this in at some point? I think you're the one to figure out how to make this happen. In a way, it's a variation of the other scrolling PR you have open, which is about making vertical scrollbars more visible: https://github.com/cockroachdb/docs/pull/6420.

**Comment by yingzhuchin** [comment]
I took a quick look. From what I can see, this will necessitate modifying the <div> tags around each SQL diagram to identify them as special components that require horizontal scroll. The diagrams currently do not have a <div> identifier. Example: 

<img width="647" alt="Screen Shot 2020-04-21 at 3 26 30 PM" src="https://user-images.githubusercontent.com/63265176/79905595-7c55c780-83e4-11ea-8487-12464af6f0e2.png">

(From https://raw.githubusercontent.com/cockroachdb/docs/master/v19.2/configure-zone.md)

In short, we go through and audit which components require horizontal scroll, then modify their <div> tags. After that we can enable horizontal scroll for said components while disabling broadly.

**Comment by luciaaaaaaa** [comment]
hi, looping in as part of redesign work. @irfansharif do you have examples of those broad components that would merit horizontal scroll? my instinct is saying to remove it all together

**Comment by irfansharif** [comment]
I don't have an authoritative list, no, but the example I screen-shotted above was from https://www.cockroachlabs.com/docs/stable/install-cockroachdb-mac.html. I believe the pattern applies broadly, so I'd second your instinct.

**Comment by jseldess** [comment]
Looks like this was fixed with the broader docs redesign. For example, on https://www.cockroachlabs.com/docs/stable/configure-zone.html#synopsis, when the screen is too narrow to show the complete diagrams, the scroll is just on those diagrams now rather than the entire body of the page. Closing, but if this is still a problem anywhere, we can reopen.

**Comment by irfansharif** [comment]
Thanks Jesse, that's great to see!
